soc/arm/silabs_exx32/efm32pg1b/Kconfig.series

```diff
@@ -18,5 +18,6 @@ config SOC_SERIES_EFM32PG1B
 	select SOC_GECKO_CMU
```

Comment:
Shouldn't this be in soc/arm/silabs_exx32/efm32pg12b/Kconfig.series?
The boards/arm/efm32pg_stk3402a/efm32pg_stk3402a.dts has #include <silabs/efm32pg12b500f1024gl125.dtsi>.

Reply:
Thank you. I think you're right. I've fixed this.